### PR TITLE
CI update - .NET 9 Preview 7

### LINKED ISSUE #9988: Update API ref for .NET 9 Preview 7
---
[Associated WorkItem - 296913](https://dev.azure.com/msft-skilling/Content/_workitems/edit/296913)

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #9988.

View by commit. The last commit ("CI update") is the docs.

### CODE REVIEW

**Comment by carlossanlop** [comment]
I see it's still in draft. Do you need any help figuring out the correctness?

**Comment by BillWagner** [comment]
@gewarren Are you expecting the warnings at this time?

**Review by carlossanlop — APPROVED** [review]

File: xml/System/MathF.xml [hunk 1]
```diff
@@ -425,32 +423,12 @@ The static fields and methods of the `MathF` class correspond to those of the <x
         <Parameter Name="x" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="y">The y coordinate of a point.</param>
-        <param name="x">The x coordinate of a point.</param>
-        <summary>Returns the angle whose tangent is the quotient of two specified numbers.</summary>
-        <returns>An angle, θ, measured in radians, such that -π ≤ θ ≤ π, and tan(θ) = <paramref name="y" /> / <paramref name="x" />, where (<paramref name="x" />, <paramref name="y" />) is a point in the Cartesian plane. Observe the following:
-
--   For (<paramref name="x" />, <paramref name="y" />) in quadrant 1, 0 &lt; θ &lt; π/2.
```
Comment (gewarren) [inline xml/System/MathF.xml:433]: @tannergooding Should this info be retained?

Comment (gewarren) [reply xml/System/MathF.xml:433]: Also in other places in this file.

Comment (gewarren) [reply xml/System/MathF.xml:433]: Will merge for now but we can put it back into the /// if desired.

**Comment by gewarren** [comment]
> @gewarren Are you expecting the warnings at this time?

I'll fix the ones that matter in the /// source.

**Comment by carlossanlop** [comment]
Weeeee let's get a new report!

**Pull request merged**